app-conf/FetcherConf.xml

```diff
@@ -32,6 +32,8 @@
   <fetcher>
     <applicationtype>mapreduce</applicationtype>
     <classname>com.linkedin.drelephant.mapreduce.MapReduceFetcherHadoop2</classname>
+    <!-- If encounter performance issues, try to use fetcher below -->
```

Comment:
@paulbramsen , I think we can do that after we test this patch in production, run it for some time, find it to be completely stable and then maybe we can add a comment saying which would be the preferred fetcher and deprecating the older one.